ecosystem/indexer/src/indexer/fetcher.rs

```diff
@@ -59,7 +59,7 @@ impl Fetcher {
 
     pub async fn run(&mut self) {
         loop {
-            if self.current_version == self.highest_known_version {
+            if self.current_version >= self.highest_known_version {
```

Comment:
FYI, this is a little confusing, i guess the current version is the current of the indexer, and highest known version is the highest version of the chain?

A comment might help saying, if we're ahead of what we know about on chain, update it accordingly.